Major doc update para seller behavior (#1258)
mercadopago/devsite-docs: guides/plugins/prestashop/integration.en.md

Once the module is installed, follow these steps to integrate:

1. Create a [seller account](https://www.mercadopago.com.ar/registration-company?confirmation_url=https%3A%2F%2Fwww.mercadopago.com.ar%2Fcomo-cobrar)  in Mercado Pago if you don’t have one yet.

> 2. Obtain the Access Token and Public Key [credentials]() and paste them into the Production and Testing fields that you will find in the module configuration.

Reviewer: ```suggestion
2. Obtain the Access Token and Public Key [credentials](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/resources/credentials) and paste them into the Production and Testing fields that you will find in the module configuration.
```

>
> Important
>

> > The [credentials]() are the keys that uniquely identify you within the platform. To operate in a **test environment** with which to simulate online payments, use the test credentials. Use **production** credentials to receive real payments.

Reviewer 2: ```suggestion
> The [credentials]() are the keys that uniquely identify you within the platform. To operate in a **test environment** in order to simulate online payments, use the test credentials. To receive real payments, use **production** credentials.
```
